packages/documentation/src/pages/getting-started/common-tasks/run-build.mdx

```diff
@@ -0,0 +1,58 @@
+# Run and build VA.gov
+
+## Prerequisite
+
+- Complete the [first](/getting-started) step in *Start up guide*.
+
+## Locally run VA.gov
+
+Start the watch task:
+```bash
+yarn watch
+```
+### Build
+  - **Metalsmith** manages this build pipeline. At a high level, this process:
+    1. retrieves and **builds** all of the **static pages** sourced from `vagov-content` or Drupal
+    2. starts **Webpack**'s devevelopment server which
+       - **builds** the **JavaScript** and **CSS** bundles, and
```

Comment:
Might tweak this later after I do the content style guide. The idea for now is that the bolded words are for people to scan for unfamiliar things. e.g. if i know what metal smith is then this should help me find the steps that i may be unfamiliar with.